### ISSUE TITLE
How many unique fields appear in the JSON columns?

### ISSUE THREAD

**Issue by jonathancstroud** [MichiganDataScienceTeam/googleanalytics#20]
Do any features appear in the training set and not the test set?
